ref: add description that geo information is derived even if IP addre… (#13828)
getsentry/sentry-docs: docs/security-legal-pii/scrubbing/server-side-scrubbing/index.mdx

In addition, we provide an [_Advanced Data Scrubbing_](/security-legal-pii/scrubbing//advanced-datascrubbing/) feature that allows more control over how filters are applied.

> ## Geo Information

Reviewer: ```suggestion
## Geographic Information
```

> Geo information is extracted based on the IP address of the user. This will happen independently of whether the setting to

Reviewer: ```suggestion
Geographic information is extracted from the user's IP address. This occurs even if the setting to stop storing IP addresses is turned on.
```

> prevent storing IP addresses is enabled.

Reviewer: ```suggestion

```
